README.md

```diff
@@ -2,8 +2,12 @@
 
-The Software Citation Tools will be a suite of tools for the scientific community to assist them in citing software that is found online and used in research and scientific papers.  The purpose of this is to reduce friction for researchers to cite software and make sure that software developers are given the credit they are due.  The suite will be developed in conjunction with Mozilla Science Lab via Abigail Cabunoc Mayes, the scientific community, and the open source software development community.  The project will require extensive interviews with members of the research community to gather information on how they are currently citing software, and how these tools can streamline that process.
+The Software Citation Tools are a package of tools written in Javascript and with the core library made available through the npm Javascript package manager. The main part of the package will be a library which will create a citation when given a link to a Github repository. Further tools will be created on top of this library to make using this library easier and extend its usefulness.
 
-At the core of the Software Citation Tools suite will be a package written in JavaScript using npm.  This package will provide functions which will take a GitHub repository URL and return a citation for the software compliant with the Software Citation Principles.  Stretch goals for this package may include accepting other types of software repositories as input, and accepting parameters that will change the format of the citation.  For example, parameters to generate long and short versions of a citation, or output for different citation standards.  The priority of stretch goals will be determined based on the needs of the scientific community.
+##About the Project
+Software Citation Tools is a project proposed and overseen by Mozilla Science Lab via Abigail Cabunoc Mayes and headed by a senior project team at The Rochester Institute of Technology's Software Engineering Department. The purpose of Software Citation Tools is to provide some tools that use the ideas shown in the [Software Citation Principles](https://www.force11.org/software-citation-principles) paper by FORCE11 to make software citation easier and more frequent in academia.
 
-Utilizing the citation package, two or more applications will be developed.  Two such applications might be a webapp interface to allow researchers to utilize the package conveniently, and a browser plugin to produce the citation when the user has navigated to a repo.
+Even though the team heading the project is doing this as a senior project, anyone is free to contribute! The RIT Software Engineering department has given approval for anyone to help out in any way without affecting the team's grade. Please feel free to contribute in any way you want!
```

Comment:
I'm not sure moving this is the right way to go about it.  Looking at how this would look on @ourobor's fork (https://github.com/Ourobor/software-citation-tools/blob/67e72a75689ed0d649423226bbe672c69f57aab1/README.md), it does draw a bit more attention to the call to action, since it's the only thing under the "Contributing" heading, but this paragraph really belongs under that heading, too.  Perhaps it would be better to bold the call to action line, or draw attention to it some other way?  I'm not sure. Thoughts?